src/sidebar/index.js

```diff
@@ -108,7 +108,7 @@ import LaunchErrorPanel from './components/LaunchErrorPanel';
 import { ServiceContext } from './service-context';
 
 // Services.
-import bridgeService from '../shared/bridge';
+import { Bridge as BridgeService } from '../shared/bridge';
```

Comment:
```suggestion
import { Bridge } from '../shared/bridge';
```

The other services have a `Service` suffix because the actual class does.